docs/developer/payments/transactions.mdx

```diff
@@ -450,6 +450,80 @@ Once `Checkout` is converted to into `Order`, the `authorizeStatus` might change
 - **For orders**: pending Transactions **are not** considered "paid" and will not be included in the [`Order.authorizeStatus`](/api-reference/orders/enums/order-authorize-status-enum.mdx#values) calculation.
 :::
 
+### TransactionItem and  TransactionEvents pspReference Matching
+
+Both [TransactionItem](/api-reference/payments/objects/transaction-item)
+and [TransactionEvent](/api-reference/payments/objects/transaction-event)
+contain a `pspReference` field, which refers to the payment service provider reference.
+	-	The `pspReference` on `TransactionItem` identifies the overall payment.
+	-	The `pspReference` on `TransactionEvent` identifies specific actions that occurred within the `Transaction`.
+
+**Key Assumptions:**
+- Only one `TransactionItem` instance can exist with a given `pspReference`.
+- For a specific `TransactionItem`, only one `TransactionEvent` of a given `type` and `pspReference` may exist, except for the following types:
+  -	`AUTHORIZATION_ACTION_REQUIRED`
+	-	`CHARGE_ACTION_REQUIRED`
+	-	`INFO`
+- Only one `TransactionEvent` of type `AUTHORIZATION_SUCCESS` is allowed to exist.
```

Comment:
Would be also good to mention that in case of modifying the authorization, need to use https://docs.saleor.io/developer/payments/lifecycle#authorization_adjustment